assets/js/routers/router.js

```diff
@@ -0,0 +1,28 @@
+var Router = Backbone.Router.extend({
+  routes: {
+    '(:tag)': 'filterLinksByTag',
+  },
+
+  initialize: function() {
+    this.template = AppTemplates.linkview;
```

Comment:
Please use views for all templating. The router will be unable to handle any events that may need to fire and is pretty poor at listening for underlying events on collections or models. If it helps you can `bower install --save marionette` and  lean on Marionette to handle rendering and/or regions.